Assets/Scripts/API/ItemsFile.cs

```diff
@@ -162,6 +162,11 @@ public struct Ingredient
         public int id;
     }
 
+    public struct JsonClone
```

Comment:
Can you add move context to describe what this is used for? `JsonClone` doesn't say much in itself. Maybe `JsonArchiveClone` would at least be more clear to someone familiar with texture archives.

Reply:
Would it make sensefor other file formats as well? JsonClone's struct could be potentially used for them as well

Reply:
Just handle the code you're adding, no need to touch other code

Reply:
What do you mean? I was wondering if the name "JsonArchoveClone" would fit other ends as well in the future, besides cloning image files

Reply:
> Just handle the code you're adding, no need to touch other code

What do you mean? I was wondering if the name "JsonArchiveClone" would fit other ends as well in the future, besides cloning image files